homeassistant/components/teslemetry/__init__.py

```diff
@@ -143,6 +157,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: TeslemetryConfigEntry) -
         if models:
             energysite.device["model"] = ", ".join(sorted(models))
 
+        # Create the energy site device regardless of direct entities so it can be used for custom service calls
```

Comment:
For example, I don't, because I have a wall connector but do not not a Powerwall or Solar, so none of the entities under this device get created for me.

Reply:
Can you extend the comment a bit explaining that so future reviewers isn't asking the same thing?